mindjunkies/courses/views.py

```
@@ -1,14 +1,13 @@
 from django.contrib import messages
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.mixins import LoginRequiredMixin
-from django.http import HttpRequest, HttpResponse
+from django.http import HttpRequest, HttpResponse, JsonResponse
```

Ruff (0.8.2)

`django.http.JsonResponse` imported but unused

Remove unused import: `django.http.JsonResponse`

(F401)